uat/testing-features/src/main/java/com/aws/greengrass/steps/MqttControlSteps.java

```diff
@@ -61,6 +62,8 @@
 @ScenarioScoped
 public class MqttControlSteps {
     private static final String DEFAULT_CLIENT_DEVICE_POLICY_CONFIG = "/configs/iot/basic_client_device_policy.yaml";
+    private static final DisconnectReasonCode DEFAULT_DISCONNECT_REASON
```

Comment:
Can use NORMAL_DISCONNECTION direcrly 

also NORMAL_DISCONNECTION can be imported statically

Reply:
fixed